src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/_params.py

```diff
@@ -0,0 +1,139 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import \
+    (register_cli_argument, register_extra_cli_argument, CliArgumentType)
+import azure.cli.core.commands.arm  # pylint: disable=unused-import
+from azure.cli.core.commands.validators import get_default_location_from_resource_group
+from azure.cli.core.commands.parameters import (
+    resource_group_name_type,
+    location_type,
+    tags_type,
+    deployment_name_type,
+    enum_choice_list,
+    get_resource_name_completion_list
+)
+from azure.cli.core._profile import Profile
+from azure.cli.core.util import get_json_object
+
+from azure.cli.core.sdk.util import ParametersContext
+
+# PARAMETER REGISTRATIONS
+
+register_cli_argument('sf cluster list', 'resource_group_name', resource_group_name_type,
+                      id_part=None, required=False, help='The resouce group name')
```

Comment:
"requiredness" is inferred from the SDK or custom command signature and does not usually need to be specified here unless the SDK is wrong for some reason.